docs/source/dev_guide/pre_release_test_unified.rst

```diff
@@ -0,0 +1,61 @@
+***************************************
+Testing directions for making a release
+***************************************
+
+Use these directions to test an E3SM Unified release candidate (RC).
+
+1. Have three shells open: one on Chrysalis, one on Compy, and one on Perlmutter. Do the following steps on each machine.
+
+2. ``cd`` to the ``zppy`` directory.
+   
+3. Check out a branch to test on. ::
+
+     git fetch upstream main
```

Comment:
Note that on `Compy`, this might not always get the latest commits from `main`. It is better to do `git fetch upstream` to fetch all branches.